resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js

```diff
@@ -77,6 +88,25 @@ const GalleryBlockSignup = ({ campaignId, path }) => {
         });
   };
 
+  const handleScholarshipButtonAnalytic = event => {
+    event.preventDefault();
+
+    if (campaignData && campaignData.signups.length) {
+      trackAnalyticsEvent(
+        'clicked_scholarship_gallery_block_view_application',
+        {
+          action: 'button_clicked',
+          category: EVENT_CATEGORIES.siteAction,
+          label: 'scholarship_gallery_card',
+          context: {
+            url: path,
+            campaignId,
+          },
+        },
+      );
+    }
```

Comment:
And per https://github.com/DoSomething/phoenix-next/pull/2448/files#r535347695, we can do something like:
```suggestion
      trackAnalyticsEvent(
        'clicked_scholarship_gallery_block_view_application',
        {
          action: 'button_clicked',
          category: EVENT_CATEGORIES.siteAction,
          label: 'scholarship_gallery_card',
          context: {
            url: path,
            campaignId,
          },
        },
      );
      
      window.location = path;
```

Reply:
ah - I see, move the place of the if statement, and use a ternary. I like the suggestions. I don't think adding `window.location = path` is needed in the place you suggested because a `href={path}` is already in `SecondaryButton`.

Reply:
Ahhh good call! So if we remove the `event.preventDefault` at the top we'll be golden 😌